R/reexports.R

```diff
@@ -11,6 +11,10 @@ generics::required_pkgs
 #' @export
 generics::tunable
 
+#' @importFrom generics tune_args
+#' @export
+generics::tune_args
```

Comment:
Same general comment about the fact that I wouldn't export `tune_args()`. But we already export `tunable()` here so maybe it is worth it to be consistent for the recipes case...